modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/BaseDDMStructureVersionCTUpgradeProcessTestCase.java

```diff
@@ -3,24 +3,20 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.dynamic.data.mapping.upgrade.v5_1_1.test;
+package com.liferay.dynamic.data.mapping.upgrade;
 
-import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
 import com.liferay.dynamic.data.mapping.model.DDMStructure;
 import com.liferay.dynamic.data.mapping.model.DDMStructureVersion;
 import com.liferay.dynamic.data.mapping.service.DDMStructureVersionLocalService;
 import com.liferay.portal.kernel.model.change.tracking.CTModel;
 import com.liferay.portal.kernel.service.change.tracking.CTService;
 import com.liferay.portal.test.rule.Inject;
 
-import org.junit.runner.RunWith;
-
 /**
- * @author Paulo Albuquerque
+ * @author Carolina Barbosa
  */
-@RunWith(Arquillian.class)
-public class DDMStructureVersionDDMValidationUpgradeProcessTest
-	extends DDMStructureDDMValidationUpgradeProcessTest {
+public abstract class BaseDDMStructureVersionCTUpgradeProcessTestCase
+	extends BaseDDMStructureCTUpgradeProcessTestCase {
```

Comment:
I think it should be BaseDDMStructureVersion**DDMStructure**CTUpgradeProcessTestCase